daemons/gptp/linux/src/linux_hal_common.cpp

```diff
@@ -213,6 +213,9 @@ void *LinuxTimerQueueHandler( void *arg ) {
 		    struct LinuxTimerQueueActionArg *arg = iter->second;
 			timerq->timerQueueMap.erase(iter);
 			timerq->LinuxTimerQueueAction( arg );
+			if( arg->rm ) {
+				delete arg->inner_arg;
+			}
 			timer_delete(arg->timer_handle);
 			delete arg;
```

Comment:
I'm going to accept this in 24 hours unless anyone else has comments. I'm sure the code is obvious to someone familiar to the code base. I was hoping you would suggest a comment along the lines of
// if rm (remove flag) is true, delete inner_arg
As a casual reader, I didn't even know rm was a flag rather than a pointer.

Sorry about asking the same question twice. I may have missed a response somewhere along the line.